lesson3/_homework/task_06/script.js

```diff
@@ -0,0 +1,22 @@
+
+//                    Adapt Academy 2019 lesson 03 - JS introduction | task 06 by Donatas Dereškevičius 
+
+
+const loadJson = () => {
+
+    fetch (`../../users.json`)
+    .then (response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error('Something went wrong ...');
+      }
+    })
+    .then(users => {
+        let usersArray = users.map((user) => `<br /> Hello from ${user.name}`);
```

Comment:
use `const` and also this will only create a new array consisting only of user names. We should keep data structure like is in users.json and only change the names.